pkg/builder/builder.go

```diff
@@ -1160,6 +1215,46 @@ func buildCanaryQuerySQL(vdb *vapi.VerticaDB) string {
 	return fmt.Sprintf("vsql %s -c 'select 1'", passwd)
 }
 
+// buildVerticaStartCommand returns the vertica start command that
+// will serve as entrypoint to the server container
+func buildVerticaStartCommand() []string {
+	startCmd := fmt.Sprintf("/opt/vertica/bin/vertica --startup-conf %s", paths.StartupConfFile)
+	return []string{
+		"bash",
+		"-c",
+		fmt.Sprintf("%s && sleep infinity", startCmd),
+	}
+}
+
+// buildNMACommand returns the command to start NMA
+func buildNMACommand() []string {
+	return []string{
+		"/opt/vertica/bin/node_management_agent",
+	}
+}
+
+// buildNMAEnvVars returns environment variables that are needed by NMA
+func buildNMAEnvVars(vdb *vapi.VerticaDB) []corev1.EnvVar {
+	if vmeta.UseNMACertsMount(vdb.Annotations) && secrets.IsK8sSecret(vdb.Spec.NMATLSSecret) {
+		return []corev1.EnvVar{
+			// Provide the path to each of the certs that are mounted in the container.
+			{Name: NMARootCAEnv, Value: fmt.Sprintf("%s/%s", paths.NMACertsRoot, paths.HTTPServerCACrtName)},
+			{Name: NMACertEnv, Value: fmt.Sprintf("%s/%s", paths.NMACertsRoot, corev1.TLSCertKey)},
+			{Name: NMAKeyEnv, Value: fmt.Sprintf("%s/%s", paths.NMACertsRoot, corev1.TLSPrivateKeyKey)},
+		}
+	}
+	return []corev1.EnvVar{
+		// The NMA will read the secrets directly from the secret store.
+		// We provide the secret namespace and name for this reason.
+		{Name: NMASecretNamespaceEnv, Value: vdb.ObjectMeta.Namespace},
+		{Name: NMASecretNameEnv, Value: vdb.Spec.NMATLSSecret},
+	}
+}
+
+func getShareProcessNamespace(vdb *vapi.VerticaDB) bool {
+	return vdb.IsSideCarDeploymentEnabled()
```

Comment:
Regarding the stdout issue with NMA and server, lets try running without this set. This may cause those to work. I assume we needed this when talking to EOM folks, but lets find the actual use case before setting it.